### PR TITLE
Fix Symfony 4 support

### PR DESCRIPTION
**Pull request by sstok**
No actual code changes were required to make this component work with Symfony 4.

### CODE REVIEW

**Comment by iltar** [comment]
2.7 is EOL in may next year. However, the plan is to drop 2.x support when we drop support for the old form handler style, so that includes 2.8.

File: composer.json [hunk 1]
```diff
@@ -14,16 +14,17 @@
...
         "hostnet/phpcs-tool":     "^4.1",
         "phpunit/phpunit":        "^5.4",
-        "symfony/debug":          "^2.7||^3.0",
-        "symfony/phpunit-bridge": "^2.7||^3.0"
+        "symfony/debug":          "^2.7||^3.0||^4.0",
+        "symfony/phpunit-bridge": "^2.7||^3.0||^4.0"
     },
-    "minimum-stability": "stable",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
```
Comment (wouterj) [inline composer.json]: Whoa this is scary. Why not only enable this when testing against Symfony 4 in travis?

Comment (iltar) [reply composer.json]: The bundle is actually using this as well: https://github.com/hostnet/form-handler-bundle/blob/master/composer.json#L48-L49

What are the possible side effects of this?

**Comment by wouterj** [comment]
> holy shit, we are supporting 3 symfony versions like this.... when is 2.7 EOL?

The core team is warned about this years ago: http://symfony.com/blog/improving-the-symfony-release-process#comment-20285 But seems like bundle maintainers aren't thought of that much sadly...

File: composer.json (hunk 1 above)
Comment (sstok) [reply composer.json]: It only affects the root package. So no real problem here.

Testing matrix could actually be improved, but I don't have time for that 😅
Thanks for the release 👍

Comment (wouterj) [reply composer.json]: This is a root-only setting and applies to all packages. So you might become dependent on a non stable version (dev, not even beta!) of a package without knowing that (as your tests all run in allow-dev mode).

Yet, as soon as you require it in an application, users might also need to add `package: @dev` to make it possible to use/install your package.

So better safe then sorry: Use strict whitelisting of dev versions (preferably beta) and only do that in the job that need it (usually, you test lower + stable + dev versions, with dev on allowed failure).

Comment (wouterj) [reply composer.json]: @sstok  Being root only is actually the problem. Add (hypothetically) `"symfony/phpunit-bridge": "4.0"` to the require section and you see what I mean (tests pass, but requiring in an app is no longer possible).

**Pull request merged**